src/api/app/models/notifications/rss_feed_item.rb

```diff
@@ -20,6 +20,18 @@ def self.cleanup
       end
     end
   end
+
+  def title
+    event.subject
+  end
+
+  def description
+    ApplicationController.renderer.new.render(
```

Comment:
You can re-use the event_mailer views from the rss feeds controller can't you? Also see my PR https://github.com/openSUSE/open-build-service/pull/3274 as it will make that easier once merged.